net2/logger.js

```diff
@@ -53,7 +53,7 @@ else if (production) {
 }
 
 function getFileTransport() {
-  let loglevel = globalLogLevel;
+  let loglevel = 'debug';
```

Comment:
We use wrapper function of logger to determine log level. loglevel here is used in winston logger itself.